node/network/bridge/src/lib.rs

```diff
@@ -861,6 +862,10 @@ where
 
 	ctx.spawn("network-bridge-network-worker", Box::pin(remote)).await?;
 
+    ctx.send_message(AllMessages::StatementDistribution(
```

Comment:
spaces in this file

Reply:
Thanks I will make a sweep searching for spaces once I am done. I still haven't found what's causing my editor to mess this up from time to time *sigh*.